guides/security/password_policy.md

```diff
@@ -1,12 +1,38 @@
-## Password Policy
-The purpose of this policy is to give guidance for the creation of strong and secure passwords following the [NIST security guidelines](https://pages.nist.gov/800-63-3/sp800-63b.html#memsecret)
+# Password and 2FA policy
+Strong password and appropriate use of 2FA (two factor authentication) are important aspects of device and software security. We all have passwords for multiple systems that are used for business related purposes and it is the important for all team members to take the appropriate steps to secure their passwords. 
 
-### Password creation
+## Password Creation
+- Passwords must be sufficiently strong such as to make cracking or guessing difficult
+- Where possible, passwords should be randomly generated (you can do this in your password manager)
+- Passwords should be more than 12 characters and contain a mixture of upper case, lower case, numbers and special characters
+- Do not use the same password for access to Made Tech systems as for any personal accounts. If you do this, and your Made Tech password is compromised, it will open your personal accounts up to vulnerablities and vice versa.
 
-- [ ] Use a secure password generator to generate complex passwords
-- [ ] Password length should be a minimum of 8 characters
-- [ ] Length is crucial because it increases entropy (number of uncertainty)
-- [ ] Never write down or share your passwords
-- [ ] Do not use a password for more than one account (reused passwords should be flagged by LastPass)
-- [ ] Password manager should follow this password policy
+## Password Management
+We use password management for Made Tech services. Individual vaults are managed by the teams who use them, and the password manager is managed by the Operations Team.
 
+Your password to access your password manager should confirm to the password policy. Biometrics is also an acceptable method of accessing the password manager.
+
+## Password Security
+
+### Obtaining Passwords via Social Engineering
```

Comment:
This feels like the least important part of this guidance, yet it takes up the most space.

What's really important?

- Be aware of someone using social engineering to gain access to your password / account
- Do not store passwords on your device outside of an encrypted password manager
- Do not write passwords down
- Be sure you're entering your password in to a legitimate website
- Passwords should generally not be shared. When these is need to share a piece of secure information such as a password do so via 1Password

Reply:
deleted this section and put the above points in